odom_navigation:
  ros__parameters:
    type_odom: false
    enable_imu: true
    velocity_alpha: 0.9

    imu_topic: "/ouster/imu"
    odom_topic: "/odom"
    pose_topic: "/current_pose"
    navigation_topic: "/agent0/navigation_data"

gps_navigation:
  ros__parameters:
    velocity_alpha: 0.9
    reference_gps_latitude: -33.722768731682656
    reference_gps_longitude: 150.6739905114244

    imu_topic: "/agent0/sensors/imu/imu/data"
    gps_topic: "/agent0/sensors/gps/gps/fix"
    navigation_topic: "/navigation_data"
